[05:00:48] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot2/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async VKBot.poll (/root/groupbot2/dist/vkbot.js:72:35)
[05:00:48] Error: {
  error_code: 10,
  error_msg: 'Internal server error: could not check access_token now, check later.',
  request_params: [Array]
}
[11:27:02] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot2/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[11:27:02] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[12:29:09] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot2/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[12:29:09] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[12:32:28] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot2/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[12:32:28] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[13:31:04] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot2/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[13:31:04] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[13:34:24] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot2/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[13:34:24] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[14:32:52] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot2/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[14:32:52] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[14:36:09] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot2/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[14:36:09] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[15:34:11] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot2/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[15:34:11] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[15:37:29] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot2/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[15:37:29] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[16:36:20] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot2/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[16:36:20] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[16:39:34] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot2/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[16:39:34] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[17:37:52] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot2/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[17:37:52] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[17:41:05] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot2/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[17:41:05] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[18:07:38] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot2/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async VKBot.poll (/root/groupbot2/dist/vkbot.js:72:35)
[18:07:38] Error: {
  error_code: 10,
  error_msg: 'Internal server error',
  request_params: [Array]
}
[18:39:18] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot2/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[18:39:18] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[18:42:34] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot2/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[18:42:34] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[19:12:55] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot2/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async VKBot.poll (/root/groupbot2/dist/vkbot.js:72:35)
[19:12:55] Error: {
  error_code: 10,
  error_msg: 'Internal server error',
  request_params: [Array]
}
[19:41:04] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot2/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[19:41:04] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[19:44:18] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot2/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[19:44:18] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[20:25:09] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot2/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async VKBot.poll (/root/groupbot2/dist/vkbot.js:72:35)
[20:25:09] Error: {
  error_code: 10,
  error_msg: 'Internal server error',
  request_params: [Array]
}
[20:42:32] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot2/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[20:42:32] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[20:45:45] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot2/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[20:45:45] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[21:01:10] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot2/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async VKBot.poll (/root/groupbot2/dist/vkbot.js:72:35)
[21:01:10] Error: {
  error_code: 10,
  error_msg: 'Internal server error',
  request_params: [Array]
}
[21:44:08] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot2/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[21:44:08] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[21:47:22] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot2/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[21:47:22] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[22:45:42] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot2/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[22:45:42] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[22:48:57] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot2/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[22:48:57] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[23:47:19] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot2/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[23:47:19] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[23:50:33] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot2/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[23:50:33] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
